data_analysis/regression.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from scipy.optimize import least_squares
+
+from get_data.coronadata import get_days, coronaData
```

Comment:
Can these imports/files/functions be renamed according to naming conventions please, so 'corona_data.py' and 'corona_data'